lib/node/rpc.js

```diff
@@ -2765,6 +2769,26 @@ class RPC extends RPCBase {
     };
   }
 
+  async getDescriptorInfo(args, help) {
```

Comment:
lets put this function after the `setloglevel` implementation (up on line 2328 or so). Down here is for helper functions